### PR TITLE
Help readers more with Web Speech API error info

### LINKED ISSUE #20749: Speech recognition getting 'language not supported' error
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/Web_Speech_API

### What specific section or headline is this issue about?

Speech recognition getting 'language not supported' error

### What information was incorrect, unhelpful, or incomplete?

I was gotten error 'language not supported' on the Edge browser. Is this problem caused by the SpeechRecognition API?

### What did you expect to see?

Information about the 'language not supported' error of the SpeechRecognition API

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/api/web_speech_api`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/Web_Speech_API
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/web_speech_api/index.md
* Last commit: https://github.com/mdn/content/commit/f7dae62645a2c735ed6f6ed63f664bf279fdfc4b
* Document last modified: 2022-09-09T05:19:45.000Z

</details>

### PR DESCRIPTION
**Pull request by sideshowbarker**
Fixes https://github.com/mdn/content/issues/20749

### CODE REVIEW

File: files/en-us/web/api/speechrecognitionerrorevent/error/index.md [hunk 1]
```diff
@@ -44,7 +44,9 @@ A string naming the type of error. The possible error types are:
   - : There was an error in the speech recognition grammar or semantic tags, or the chosen
     grammar format or semantic tag format was unsupported.
 - `language-not-supported`
-  - : The language was not supported.
+  - : The user agent does not support the language specified in the value of [`lang`](/en-US/docs/Web/API/SpeechRecognition/lang) attribute of the {{domxref("SpeechRecognition")}} object.
+
```
Comment (bsmth) [inline files/en-us/web/api/speechrecognitionerrorevent/error/index.md:48]: style question: do you want the extra line breaking this paragraph?

File: files/en-us/web/api/speechsynthesiserrorevent/error/index.md [hunk 2]
```diff
@@ -50,6 +50,8 @@ A string containing an error code. Possible codes are:
 - `language-unavailable`
   - : No appropriate voice was available for the language set in
     {{domxref("SpeechSynthesisUtterance.lang")}}.
+
```
Comment (bsmth) [inline files/en-us/web/api/speechsynthesiserrorevent/error/index.md:53]: Same again here, I think keeping the paragraph together is easier to read, but that's a personal preference.

File: files/en-us/web/api/speechrecognitionerrorevent/error/index.md (hunk 1 above)
Comment (sideshowbarker) [reply files/en-us/web/api/speechrecognitionerrorevent/error/index.md:48]: Yeah, it was intentional. But if you think it'd be better without the linebreak, then let's do it that way instead

**Review by bsmth — APPROVED** [review]
Looks good, thanks. Two minor comments on style, but ready to merge after considering 👍🏻

File: files/en-us/web/api/speechrecognitionerrorevent/error/index.md (hunk 1 above)
Comment (bsmth) [reply files/en-us/web/api/speechrecognitionerrorevent/error/index.md:48]: I think it's better without the linebreak, we have one para consolidated for the explanation, what do you think?

image

Comment (sideshowbarker) [reply files/en-us/web/api/speechrecognitionerrorevent/error/index.md:48]: > I think it's better without the linebreak, we have one para consolidated for the explanation, what do you think?

I think that in general, having one paragraph consolidated for list items in associative lists is not a goal. We have other existing articles with associative lists that have multiple paragraphs per list item.

But I don’t feel strongly about it in this case — if we you feel it’d be better in this case, then yeah let’s change it.

Comment (bsmth) [reply files/en-us/web/api/speechrecognitionerrorevent/error/index.md:48]: Sure, I just find it more readable. I'm fine with either way, I'll leave it your call whether to change or merge 🙌🏻

Comment (sideshowbarker) [reply files/en-us/web/api/speechrecognitionerrorevent/error/index.md]: > Sure, I just find it more readable. I'm fine with either way, I'll leave it your call whether to change or merge 🙌🏻

👍 Went ahead and closed them up. It does read better here 😄

Comment (bsmth) [reply files/en-us/web/api/speechrecognitionerrorevent/error/index.md]: Nice one, thanks 🎉

**Pull request merged**